CryptoWorld.News/CryptoWorld.News.Core/Services/News/NewsService.cs

```diff
@@ -0,0 +1,128 @@
+﻿using AngleSharp;
+using CryptoWorld.News.Data;
+using CryptoWorld.News.Data.Models;
+using CryptоWorld.News.Core.Interfaces;
+using CryptоWorld.News.Core.ViewModels.Home_Page;
+using System.Globalization;
+using System.Text;
+
+namespace CryptоWorld.News.Core.Services.News
+{
+    public class NewsService : INewsService
+    {    
+        private readonly AngleSharp.IConfiguration config;
+        private readonly IBrowsingContext context;
+        private List<PageNewsModel> homeNews;
+        private List<string> urls;
+        private readonly ApplicationDbContext dbContext;
+        public NewsService( ApplicationDbContext _dbContext)
+        {
+            config = Configuration.Default.WithDefaultLoader();
+            context = BrowsingContext.New(config);
+            homeNews = new List<PageNewsModel>();
+            urls = new List<string>();
+            dbContext = _dbContext;
+        }
+
+        public async Task<List<PageNewsModel>> HomePageNews(int pagesCount = 7)
+        {
+
+            for (int i = 1; i <= pagesCount; i++)
+            {
+                var document = await context.OpenAsync($"https://money.bg/finance?page={i}");
```

Comment:
Could you please add this URL to the configurations? We might want to update it and if the app is running we would need to stop it. That's why it's better to add this into a configuration (currently appsettings.json)